LinkIt-Experiments/Updates_11Apr

```diff
@@ -0,0 +1,4 @@
+Results
```

Comment:
I meant you need to fill this file with the results that you had posted with your PR about testing(preferably as bullets). It will keep a documentation about what tests we did and what results we got.

Same for issues below, put issues that we faced here, for eg the readings we got weren't consistent.